Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -184,381 +206,57 @@ void MonteCarloAbsorption::exec() {
       } else {
         // elastic case already initialized
       }
-      doSimulation(detPos, lambdaIn, lambdaOut, yValues[bin], eValues[bin]);
+      std::tie(signal[j], std::ignore) =
```

Comment:
I'm not sure to be honest. The original author of the algorithms is not around and it makes no mention of the errors anywhere.